app/assets/javascripts/components/common/text_area_input.jsx

```diff
@@ -24,10 +24,13 @@ const TextAreaInput = ({
   markdown,
   className,
   clearOnSubmit,
-  invalid
+  invalid,
+  label,
+  spacer
```

Comment:
What is this for? It doesn't look like this property gets used in any components that use TextAreaInput.

Reply:
a custom label that can be be given to text_area component. Sir, we are using this in our modals of 'creating library', 'creating categories' and 'adding modules'. 
Whenever there is any error like no text provided by user, or exceeded the max limit of characters, then along with the boundaries of text_area_input.jsx this label also becomes red.
Eg :
![image](https://github.com/user-attachments/assets/6b392417-fd47-4546-bf11-c1fe2f56d08c)